src/components/Slideshow.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Slide } from "react-slideshow-image";
+import "react-slideshow-image/dist/styles.css";
+import "../styles/slideshow.css";
+
+const spanStyle = {
+  padding: "10px",
+  background: "rgba(0,0,0,0.2)",
+  color: "white",
+  position: "absolute",
+  bottom: "8px",
+};
+
+const divStyle = {
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  backgroundSize: "cover",
+  height: "500px",
+};
```

Comment:
If at all possible its best not to use inline styles like this, instead putting the styles all in the CSS files